test/integration/relations.test.js

```diff
@@ -86,6 +87,106 @@ describe('Relations', function () {
     describe('Many-to-Many', function () {
         before(utils.db.init('relations1'));
 
+        describe('EQUALS', function () {
+            it('tags.slug equals "animal"', function () {
+                const mongoJSON = {
+                    'tags.slug': 'animal'
+                };
+
+                const query = makeQuery(mongoJSON);
+
+                return query
+                    .select()
+                    .then((result) => {
+                        result.should.be.an.Array().with.lengthOf(3);
+                    });
+            });
+        });
+
+        describe('Multiple where clauses for relations', function () {
+            it('tags.slug equals "animal" and posts_tags.sort_order is 0 and featured is true', function () {
+                // where primary tag is "animal"
+                const mongoJSON = {$and: [
```

Comment:
Out of curiosity, is this the query we generate in Ghost? Could you point to some test case where I could find a sample?

Reply:
https://github.com/TryGhost/Ghost/issues/10105#issuecomment-438386437

Reply:
si, claro que! :ok_hand:

Reply:
I think we can leave this as a sample to show that more complex structures work. Checked what exatly is genrated by nql-lang for the case from a comment:
```
                    parse('tags.slug:en+visibility:public+posts_tags.sort_order:0').should.eql(
                        {
                            $and: [
                                {'tags.slug': 'en'},
                                {visibility: 'public'},
                                {'posts_tags.sort_order': 0}
                            ]
                        }
                    );
```
So the structured is not nested in our situation